app/models/decidim/ludens/participative_action.rb

```diff
@@ -21,7 +21,7 @@ def self.actions_file
         @actions_file ||= if File.exist?(Rails.root.join("config/participative_actions.yaml"))
                             YAML.safe_load(File.read(Rails.root.join("config/participative_actions.yaml")))
                           else
-                            YAML.safe_load(File.read("#{`bundle info decidim-ludens --path`.strip}/config/participative_actions.yaml"))
+                            YAML.safe_load(File.read(decidim_ludens_path.join("config/participative_actions.yaml")))
```

Comment:
`decidim_ludens_path` can be nil here, it can causes crash when call `join()` on Nil Object